src/CoreBundle/Model/Metadata.php

```diff
@@ -60,6 +65,7 @@ public function __construct($title, $description = null, $image = null, $domain
+        $this->isImageAvailable = null !== $image;
```

Comment:
No need to store this in a field IMO: computing it every time should be performant enough

Reply:
Yes, but as image will be initialized with default value, how to know if it's default or not after initialization? To check image value against default one will be less performant I guess...

Reply:
You can store the default value in a constant and do the comparison against it, performance will really not be an issue here IMO.

Reply:
Done